test/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests/Listener/ResponseHeaderTests.cs

```diff
@@ -74,12 +81,18 @@ public class ResponseHeaderTests
 
                 HttpResponseMessage response = await responseTask;
                 response.EnsureSuccessStatusCode();
-                Assert.Equal(3, response.Headers.Count());
-                Assert.True(response.Headers.TransferEncodingChunked.Value);
+                Assert.Equal(2, response.Headers.Count());
+                Assert.False(response.Headers.TransferEncodingChunked.HasValue);
                 Assert.True(response.Headers.Date.HasValue);
                 Assert.Equal("Microsoft-HTTPAPI/2.0", response.Headers.Server.ToString());
                 Assert.False(response.Content.Headers.Contains("Content-Length"));
                 Assert.Equal(0, response.Content.Headers.Count());
+
+                // Send a second request to check for connection corruption.
```

Comment:
nit: change the comment to "to check that the connection wasn't corrupted." I was here wondering why the connection would be corrupted and how that code was checking that 😝